package/src/components/Message/hooks/useMessageActions.tsx

```diff
@@ -291,18 +291,20 @@ export const useMessageActions = <
                       [
                         {
                           onPress: () => setOverlay('none'),
-                          text: t('Dismiss'),
+                          text: t('Ok'),
                         },
                       ],
                     );
                   } catch (err) {
```

Comment:
Won't the linter complain that this error is unused? o_O 

Anyways, should we log this to the console somehow maybe?

Reply:
I don't think so, as this is anyways an alert, and the user is getting feedback about it. And if we really need it we can have it in a different PR as this would have to be implemented for other actions too.

Reply:
I think that if we don't use the error, we should just remove the argument entirely, and have a bit "[boy scout mentality](https://www.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)" about it, but if you don't feel like adding it here, mind adding it to our new Trello board to look into later?